src/demos/calcite-shell-full-window.html

```diff
@@ -541,6 +541,14 @@ <h3 class="heading" slot="header-content">WE CARE A LOT</h3>
           <calcite-pick-list-item text-label="about the NASA shuttle falling in the sea" value="2"></calcite-pick-list-item>
           <calcite-pick-list-item text-label="about starvation and the food that Live Aid bought" value="3"></calcite-pick-list-item>
           <calcite-pick-list-item text-label="about disease, baby rock, Hudson, rock," value="4"></calcite-pick-list-item>
+          <calcite-pick-list-item text-label="about disease, baby rock, Hudson, rock," value="4"></calcite-pick-list-item>
```

Comment:
Nitpick: Should these be different? If not, can the label be updated to reflect why they're repeated?

Reply:
Updated with lorem ipsum